### ISSUE TITLE
No shortcut seen in Start menu after installation of DB Browser for SQLite

### ISSUE THREAD

**Issue by ganeshkamath89** [sqlitebrowser/sqlitebrowser#2575]
After installation of DB Browser for Sqlite, I am not seeing any folder in Start Menu options.

It might help to provide an option in Start Menu because usually people who don't install a shotcut on the screen rely on a shortcut to be avaialbe in start menu options on Windows.

OS: Windows 10 1909 Build 18363
Version: 3.12.1

**Comment by chrisjlocke** [comment]
When you say 'installed', what did you install?
If the .msi version, did you see this screen?

![image](https://user-images.githubusercontent.com/7872218/106909204-f4814580-66f7-11eb-9bac-55c1eb52b2b0.png)

It may have not have appeared if you were upgrading from a previous version.
Or, if you downloaded the .ZIP, extracted, and ran the .exe file directly (which I tend to do).

**Comment by ganeshkamath89** [comment]
Okay. You might have made this decision conciously.
But in my opinion:
1) Give the Desktop shortcut as option - this will avoid clutter on desktop if they unselect
2) Don't give the option in Program Menu
3) When I type "DB Browser for Sqlite" in Windows Start menu search, I am not getting any result. I think it needs to be registered in the registry. I am not aware how other applications do it

**Comment by chrisjlocke** [comment]
1. The desktop shortcut is an option - the user can check or uncheck the option.
2. An option is needed here to place an option in the start menu.
3. Unsure - it works for me (Windows 10)

![image](https://user-images.githubusercontent.com/7872218/106912971-74f57580-66fb-11eb-9bf4-3fbfd170fc69.png)

**Comment by chrisjlocke** [comment]
@ganeshkamath89 - Did you install the .msi or .zip version?
Did you see the screen I posted above?

**Comment by ganeshkamath89** [comment]
I installed the .msi but when I search in start menu, it only shows me the installer file, it does not show the application

**Comment by chrisjlocke** [comment]
OK, but in %appdata%\Microsoft\Windows\Start Menu\Programs, do you have a 'DB Browser for SQLite' folder?

**Comment by ganeshkamath89** [comment]
checked the folder. It is not present there.

**Comment by villanueval** [comment]
I'd like to add my vote for a default behavior in the installer so that it creates a shortcut in the Start menu. It is the expected behavior after installing a program. Otherwise, it looks like something went wrong and there is no clear way on how to find the installed program.

[The Carpentries](https://carpentries.org/) is using DB Browser to teach people SQL. Most users won't be able to find the program after installation. 

In addition, "Program Menu" is not clear if it is the same as the Start menu. I wasn't sure, so novice users won't know what this means.

**Comment by justinclift** [comment]
Yeah, good point.  It's probably about time we start looking into the installer pieces again, improving things, etc.

**Comment by Hitaishisavanur** [comment]
same here with 64 bit version

**Comment by etshy** [comment]
just installed DB Browser and the problem is still present.
I didn't check any checkboxes  but it still should be searchable in Windows menu.

I'm on Windows 11 23H2

**Comment by nadiahmdi** [comment]
First I have installed DB.Browser.for.SQLite-3.12.2-win64 on Windows 11 without checking the options for showing on desktop or start menu and I had the same problem. I couldn't find it with a search in the start menu. So then I installed it by checking the options for appearing on the desktop and start menu; this time it works. I think it is better to check the options during installation.

**Comment by chrisjlocke** [comment]
> I didn't check any checkboxes but it still should be searchable in Windows menu.

If you didn't add any shortcuts, what would it search for?  The search in Windows only finds shortcuts on your desktop, etc.  It doesn't search for every file.

If you want a proper search engine for Windows, install 'Everything'
https://www.voidtools.com/

**Comment by webprofusion-chrisc** [comment]
Hi, it would be great if the Start Menu shortcut was enabled by default, I knew I had the app installed but had to go looking in program files for it, eventually uninstalled and re-installed.

Most windows users expect a start menu entry _by default_, not by exception. Desktop shortcuts are definitely less popular but I'd go as far as saying that installing without a Start Menu entry is a user experience bug.

**Comment by justinclift** [comment]
Yep, you're definitely correct.  Getting that sorted just kind of ahhh.... fell off the list of tasks we're working on. :grimacing:

We should probably look into it again, as from rough memory it didn't seem like it would be super hard to get working.

Also from rough memory, I *think* it was something we could accomplish using the WiX Toolset that we already use for creating Windows packages.  There might even be code in an existing PR around to make it happen (unsure).

**Comment by webprofusion-chrisc** [comment]
@justinclift I think (but don't know for sure) that you need `Value="1"` for this property (which is then referenced in the UI file): https://github.com/sqlitebrowser/sqlitebrowser/blob/e49934fcea5c8da65ba0a5649497b603bf763065/installer/windows/product.wxs#L234

**Comment by justinclift** [comment]
Cool.  Hopefully it'll turn out to be that simple.  @lucydodo @karim Any interest in looking at this? :smile:

**Comment by lucydodo** [comment]
@justinclift @webprofusion-chrisc Thank you, the shortcut is now installed by default in the program menu.
This change will be included in the next release, v3.13.1. Until then, please use the [continuous](https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous), [nightly](https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/nightly) build.

> Tested in Windows Sandbox environment.

**Comment by justinclift** [comment]
AWESOME! :grin:
